demos/camassa-holm/camassaholm.py.rst

```diff
@@ -206,7 +206,7 @@ Finally, we check if it is time to dump the data. ::
      dumpn += 1
      if dumpn == ndump:
         dumpn -= ndump
-        ufile << u1
+        ufile.write(u1, time=t)
```

Comment:
There was previously some magic format you could pass the data in to `<<` that allowed you to specify the timestep.  But I don't think anyone knew what it was, or used it.